savoirfairelinux/sous-chef
# Lingxiaoyang maintenance/20170717

## Fixes # by @lingxiaoyang 

### Changes proposed in this pull request:

* This is the continuation of the work started by @lingxiaoyang in the PR #781 

### Status


### How to verify this change

Step 1: From your project repository, check out a new branch and test the changes.

git checkout -b erozqba-lingxiaoyang-maintenance/20170717 dev
git pull https://github.com/erozqba/sous-chef.git lingxiaoyang-maintenance/20170717

Step 2: Recreate the web service

docker-compose up -d --no-deps --build web

Step 3: Test the changes with DEV settings

Go to sous_chef http://0.0.0.0:8000 and add/edit/remove some avatars.

Step 4: Test the changes prod PROD settings

docker-compose -f docker-compose.yml -f docker-compose.prod.yml -up -d 
Go to sous_chef and add/edit/remove some avatars. 

### Deployment notes and migration


### New translatable strings


### Additional notes

*If applicable, explain the rationale behind your change.*
